APIKit.podspec

```diff
@@ -10,6 +10,12 @@ Pod::Spec.new do |s|
 
   s.ios.deployment_target = "8.0"
   s.osx.deployment_target = "10.9"
+  if s.respond_to?(:watchos)
+    s.watchos.deployment_target = "2.1"
```

Comment:
Oh, my bad. 2.1 might be set because of implicit deployment target in Xcode 7.2 because Result 1.0.0 does not have explicit deployment target for watchOS. I updated Result to 1.0.1 in #111 and set  WATCHOS_DEPLOYMENT_TARGET 2.0, but overlooked this :skull: